pkg/sql/logictest/testdata/logic_test/builtin_function

```diff
@@ -1388,6 +1388,13 @@ select ln(4.0786335175292462e+34::decimal)
 ----
 79.693655171940461633
 
+query IB
+SELECT LENGTH(GEN_RANDOM_UUID()),
+       GEN_RANDOM_UUID() = GEN_RANDOM_UUID()
+----
+16
+false
+
```

Comment:
nit: I know it doesn't match the one right below here, but I think this test is short enough that the query and expected results can all go on one line.

Reply:
Done.